doc/examples/fiber_to_bundle_coherence.py

```diff
@@ -0,0 +1,280 @@
+# -*- coding: utf-8 -*-
```

Comment:
Isn't the same figure already available at http://nipy.org/dipy/_images/stochastic_process.jpg??
Please delete the figure from this PR. Although tutorials can generate pictures you cannot add external pictures in the examples folder. Please use the existing picture as you did in the other example!
Thanks in advance!